Makefile.local

```diff
@@ -8,12 +8,19 @@ ifeq ($(UNAME), Linux)
     ANDROID_CC=$(ANDROID_NDK)/toolchains/llvm/prebuilt/linux-x86_64/bin/aarch64-linux-$(ANDROID_SDK_VERSION)-clang
     ANDROID_CFLAGS=--target=aarch64-linux-$(ANDROID_SDK_VERSION)
     ANDROID_RANLIB=$(ANDROID_NDK)/toolchains/llvm/prebuilt/linux-x86_64/bin/aarch64-linux-android-ranlib
+else
+ifeq ($(UNAME), Darwin)
+    ANDROID_NDK=~/Library/Android/sdk/ndk/$(ANDROID_NDK_VERSION)
```

Comment:
Could you try something for me? I'm considering adding this for the Linux and Windows builds: https://github.com/PJayB/ioq3quest/blob/linux-build-improvements/Makefile.local. This will allow folks to customize their SDK location without making local changes, but should still work for everyone who has installed to the default location.

Does the following work on MacOS for you?
```
ifeq ($(UNAME), Darwin)
    ANDROID_SDK_ROOT?=$(HOME)/Library/Android/sdk
    ANDROID_NDK=$(ANDROID_SDK_ROOT)/ndk/$(ANDROID_NDK_VERSION)
```

Tests:
* no `ANDROID_SDK_ROOT` set, Android SDK installed to default location, should still build
* `export ANDROID_SDK_ROOT=<some other location>`, Android SDK not installed to default location, should pick up new location and build

If this is too much bother, no worries, I'll be able to test later, but it would certainly help me out if you are willing to try it.

Reply:
Sorry, I just saw this message. I'd be happy to give it a try tomorrow!

Reply:
Yep, it works! I think that's a good approach. I was going to raise using `ANDROID_HOME` instead, but it turns out it's [been deprecated](https://developer.android.com/studio/command-line/variables) in favour of `ANDROID_SDK_ROOT`, so I think you're spot on.

Reply:
Thanks @ceno!